### ISSUE TITLE
Suggestion - No reversing with temporary waypoints option

### ISSUE THREAD

**Issue by qtbear** [Courseplay/courseplay#5859]
Hello,

I believe it would be quite helpful to have an option that disables the temporary waypoints from making reversing maneuvers.

Reversing with a trailer or multiple trailers during temporary waypoints often break so preventing a reversing maneuver and force only going forward would enable the use of setups that were not possible before.

Thank you for considering this

**Comment by Tensuko** [comment]
with one trailer no problem, with more trailers ofc a problem, no one does this.
Won't add an option for this.

**Comment by excubated** [comment]
@Tensuko sorry if I'm overstepping here, but I have experienced CP deciding to 3-point turn a chain of implements several times, also with "Use reversing for pathfinding" turned off.

Maybe most spectacularly with the Borgault 24m seed drill+air tank train. Both the seeder and the tank has A-frame drawbars with turning axles, and four joints on the hitch isn't really doable in reverse 😅

I've found a decent workaround, just have to generate course with 2 headlands and manually overriding the tractor's turning circle to something realistic. If i leave it on auto it will for some reason increase to well past 35m, even with tracked tractors.
